src/services/WalletService/deterministic/mnemonic.ts

```diff
@@ -1,6 +1,6 @@
 import { default as Wallet } from 'ethereumjs-wallet';
 
-import { decryptMnemonicToPrivKey } from '@services/EthService';
```

Comment:
Can you explain by which criteria you have extracted to `@utils`? This seems like something the `EthService` should do?

Reply:
Extracted out shared utils (used in multiple places), and utils that aren't used in the EthService directory.